docs/graphql/core/deployment/deployment-guides/nhost-one-click.rst

```diff
@@ -23,38 +23,33 @@ One-click deploy on Nhost
 .. note::
    Make sure to `create an Nhost account <https://nhost.io/register>`__ first. Nhost offers a 30-day trial with its starter plan.
 
-Once you're logged into your Nhost account, click the button bellow to configure your project.
+Once you're logged into your Nhost account, click the ``create project`` button below to configure your project.
 
-.. image:: https://nhost.io/images/create-project-button.png
-   :width: 300px
-   :class: no-shadow
-   :alt: Deploy with Nhost
+.. thumbnail:: /img/graphql/core/deployment/create-project-nhost.png
```

Comment:
Are you comfortable with all your personal information showing here:

![image](https://user-images.githubusercontent.com/5722022/97962753-1b0ca900-1db6-11eb-853e-687acef63386.png)

What we usually do is blur the image and the name. This can be easily done with Skitch that you could easily use now that you have a mac :)

Reply:
I will do just that.